frontend/src/pages/about/PageOutreach.vue

```diff
@@ -0,0 +1,146 @@
+<!-- 
+    This is a page for information about outreach and community engagement.
+ -->
+
+<template>
+  <AppSection>
+    <AppHeading>Outreach</AppHeading>
+    <p>
+      The Monarch Initiative is a community effort, and we welcome your
+      participation!
+    </p>
+    <AppSection>
+      <AppHeading>Ways to Get Involved</AppHeading>
+
+      <ul>
+        <li>
+          <a
+            href="https://oboacademy.github.io/obook/courses/monarch-obo-training/"
+            target="_blank"
+            >OBO Academy: Monarch Training Series</a
+          >
+          <p>
+            Training for Semantic Engineers, Ontology Curators, and Knowledge
+            Scientists on basic technologies in and around the Monarch
+            Initiative and Open Biological and Biomedical Ontologies (OBO).
+          </p>
+          <ul>
+            <li>
+              The OBO Academy develops Self-paced tutorials in the OBOOK, hosts
+              fortnightly live training sessions on Zoom, and maintains a
+              YouTube channel with lesson recordings.
+            </li>
+            <li>
+              Youtube channel:
+              <a href="https://www.youtube.com/@obo-academy" target="_blank"
+                >https://www.youtube.com/@obo-academy</a
+              >
+            </li>
+            <li>
+              Join the OBO Academy mailing list using
+              <a href="https://groups.google.com/g/obo-academy" target="_blank"
+                >https://groups.google.com/g/obo-academy</a
+              >
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="https://mondo.monarchinitiative.org" target="_blank"
+            >The Mondo Disease Ontology (Mondo)</a
+          >
+          <p>
+            Mondo aims to harmonize disease definitions across the world. The
+            name Mondo comes from the latin word ‘mundus’ and means ‘for the
+            world.’
+          </p>
+          <ul>
+            <li>
+              The Mondo Outreach calls aim to bring together stakeholders and
+              clinical terminology experts to discuss specific use cases and
+              requirements. Our goal is to improve Mondo with a focus on our
+              user needs.
+            </li>
+            <li>
+              These calls are scheduled for the second Thursday of every month.
+              You can find more information about the calls
+              <a
+                href="https://mondo.monarchinitiative.org/pages/workshop/#outreach"
+                target="_blank"
+                >here</a
+              >.
```

Comment:
Please make the additional information about the Mondo Outreach calls sub-bullets to the original description.